pkg/solana/fees/utils.go

```diff
@@ -64,14 +64,15 @@ func ParseBlock(res *rpc.GetBlockResult) (out BlockData, err error) {
 		// filter out consensus vote transactions
 		// consensus messages are included as txs within blocks
 		if len(baseTx.Message.Instructions) == 1 &&
+			len(baseTx.Message.AccountKeys) > int(baseTx.Message.Instructions[0].ProgramIDIndex) &&
```

Comment:
Can you add a comment above explaining what this check means? It's a little unclear to me here,

Reply:
Just added some comments. It was just an input validations the auditors suggested in case we get malformed transactions from an RPC